tests/virtualization/xen/virtmanager_init.pm

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: This test connects to hypervisor and check our VMs
+# Maintainer: Pavel Dostál <pdostal@suse.cz>
+
+use base "x11test";
+use xen;
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+    select_console 'x11';
+    my $hypervisor = get_required_var('QAM_XEN_HYPERVISOR');
+
+    x11_start_program 'virt-manager', target_match => 'virt-manager';
+
+    assert_screen 'virt-manager_add-connection';
+    send_key 'spc';
+    send_key 'down';
+    send_key 'down';
+    send_key 'spc';
+    save_screenshot;    # XEN selected
```

Comment:
save_screenshot is usually slow, for example
```perl
type_string "hello world";
save_screenshot;
```
will make screenshot `hello w`

You can use `wait_still_screen 1` instead of make_screenshot - it's more reliable if you want to see actual screen.